lib/db/init.test.ts

```diff
@@ -0,0 +1,39 @@
+import { initDB } from "./init";
+import {db} from "./";
+
+describe("initializes database", () => {
+
+    it("returns true", () => {
+        expect(initDB()).toBeTruthy();
+    });
+    
+    it("populates buildings and lockers", () => {
+        const ELW_COUNT = 309;
+        const ECS_COUNT = 529;
+        const ELW_ID = 1;
+        const ECS_ID = 2;
+
+        const buildings = db.prepare("SELECT * FROM buildings").all();
+        expect(buildings[0].name).toBe("engineering lab wing");
+        expect(buildings[1].name).toBe("engineering computer science");
+
+        const elwLockers = db.prepare("SELECT num FROM lockers WHERE building_id=" + ELW_ID)
```

Comment:
What you've just done is your first SQL injection vulnerability 🎉 
While this won't be an issue here imagine `ELW_ID` being a user input and what the entire statement could become as a result.

Reply:
I thought `prepare` is always a single statement? Otherwise use `exec`. I see your point though, and I've changed it.